model-optimizer/mo/utils/ir_reader/restore_graph.py

```diff
@@ -64,7 +90,6 @@ def save_restored_graph(graph: Graph, path: str, meta_data, name=None):
         PackBinaryWeights,
         BlobNormalizer,
         ConvolutionNormalizer,
-        KaldiRemoveMemoryOutputBackReplacementPattern,
```

Comment:
@achetver, why is it removed?

Reply:
This transformation performs outputs removing and we can use  `node_normalize_outputs` function and common mechanism to do it. Also `assign_add_output_result` function removed for this reason too.